### ISSUE TITLE
[Calculator][Android/iOS] History icon is gone on Programmer calculator when changing the orientation of the screen

### ISSUE THREAD

**Issue by Hpancha** [unoplatform/uno#2158]
### **Steps To Reproduce:**

1. Launch an UNO Calculator app on Android device.
2. Open **Programmer** calculator.
3. Observe the History icon located on the top right corner.
4. Now change the orientation to landscape mode and then again change it to the portrait mode.
5. Now observe the history icon again.

### **Actual behavior:**

- The icon is missing after changing the orientation of the app.

### **Expected behavior:**

- User can see the history icon even after changing the orientation of the app.

### **Attachment:**

[Missing_icon.zip](https://github.com/unoplatform/calculator/files/3881124/Missing_icon.zip)

**Before changing the orientation:**
![IMG_0324](https://user-images.githubusercontent.com/51453677/69455006-77a5be00-0d35-11ea-9a93-551a49e14982.PNG)

**After changing the orientation:**
![IMG_0323](https://user-images.githubusercontent.com/51453677/69455037-82f8e980-0d35-11ea-9b0c-bf26be6d073a.PNG)

### **Device and Application Information:**
 - **OS Build:** ios and Android
 - **Build Version:** 1.2.0-uno.558
 - **Devices:**  iPhone XR (13.2), Samsung Note 10+ (9)

### **Note:**
- Refer the attached video and images to get better idea.

**Comment by jeromelaban** [comment]
Good catch! It should not be here in the first place (memory is not available in programmer mode)

**Comment by sakshi173** [comment]
The issue is Reproducible.

**Comment by sakshi173** [comment]
The issue is still present.

**Comment by MartinZikmund** [comment]
No longer relevant to Skia renderer targets
